com.gooseberrydevelopment.pinepods.yml

```diff
@@ -0,0 +1,41 @@
+id: com.gooseberrydevelopment.pinepods
```

Comment:
do you have your github  https://github.com/madeofpendletonwool/ linked somewhere on https://gooseberrydevelopment.com/#about ? I need to make a connection between the two to verify.

Reply:
Yes, there's a github link right on the front of the main site on the bottom left. 
![image](https://github.com/user-attachments/assets/4844df7d-6400-4c81-b326-3a4ab2170261)
So at [https://gooseberrydevelopment.com](https://gooseberrydevelopment.com/#about)